src/Compilers/CSharp/Test/Symbol/Symbols/Retargeting/RetargetingTests.cs

```diff
@@ -892,6 +1003,11 @@ protected override void CheckSymbolsInternal(Symbol a, Symbol b)
         {
             a = a.OriginalDefinition;
             b = b.OriginalDefinition;
+            if (a is FunctionPointerTypeSymbol && b is FunctionPointerTypeSymbol)
+            {
+                return;
+            }
```

Comment:
Why do we need to skip the check here for `FunctionPointerTypeSymbol`?

Reply:
Because function pointers don't have underlying symbols, it's a meaningless check.

Reply:
If we had pointer symbol tests in this file, we'd have to bail out for them as well.

Reply:
Consider adding a comment here